torch/_inductor/config.py

```diff
@@ -1031,6 +1031,11 @@ class halide:
     scan_kernels = False
 
 
+class simplefsdp:
```

Comment:
later on we can have another config, e.g. `bucket_strategy` and choose from `transformer_block` vs. `greedy` vs. `dynamic_programming`, etc.

Reply:
sure, sg